README.md

```diff
@@ -150,16 +150,83 @@ A device attribute MUST be one of these:
     <td>Yes</td>
   </tr>
   <tr>
-    <td>$stats/uptime</td>
+    <td>$fw/name</td>
     <td>Device → Controller</td>
-    <td>Time elapsed in seconds since the boot of the device</td>
+    <td>Name of the firmware running on the device. Allowed characters are the same as the device ID</td>
+    <td>Yes</td>
+    <td>Yes</td>
+  </tr>
+  <tr>
+    <td>$fw/version</td>
+    <td>Device → Controller</td>
+    <td>Version of the firmware running on the device</td>
     <td>Yes</td>
     <td>Yes</td>
   </tr>
   <tr>
+    <td>$nodes</td>
+    <td>Device → Controller</td>
+    <td>
+      Nodes the device exposes, with format <code>id</code> separated by a <code>,</code> if there are multiple nodes.
+    </td>
+    <td>Yes</td>
+    <td>Yes</td>
+  </tr>
+  <tr>
+    <td>$implementation</td>
+    <td>Device → Controller</td>
+    <td>An identifier for the Homie implementation (example <code>esp8266</code>)</td>
+    <td>Yes</td>
+    <td>Yes</td>
+  </tr>
+  <tr>
+    <td>$implementation/#</td>
+    <td>Controller → Device or Device → Controller</td>
+    <td>You can use any subtopics of <code>$implementation</code> for anything related to your specific Homie implementation.</td>
+    <td>Yes or No, depending of your implementation</td>
+    <td>No</td>
+  </tr>
+  <tr>
     <td>$stats/interval</td>
     <td>Device → Controller</td>
-    <td>Interval in seconds at which the <code>$stats/+</code> are refreshed</td>
+    <td>Interval in seconds at which the <code>$stats/+</code> are refreshed: See next section for details about statistical attributes</td>
+    <td>Yes</td>
+    <td>Yes</td>
+  </tr>
+</table>
+
+For example, a device with an ID of `686f6d6965` with an `outdoor-probe` and a `windsensor` node would send:
+
+```java
+homie/686f6d6965/$homie → "2.1.0"
+homie/686f6d6965/$name → "Weather station"
+homie/686f6d6965/$localip → "192.168.0.10"
+homie/686f6d6965/$mac → "DE:AD:BE:EF:FE:ED"
+homie/686f6d6965/$stats/uptime → "120"
+homie/686f6d6965/$stats/interval → "60"
+homie/686f6d6965/$fw/name → "weatherstation-firmware"
+homie/686f6d6965/$fw/version → "1.0.0"
+homie/686f6d6965/$nodes → "outdoor-probe,windsensor"
+homie/686f6d6965/$implementation → "esp8266"
+homie/686f6d6965/$online → "true"
+```
+
+### Device statistics
```

Comment:
Ah I see... However look at this TOC:

![grafik](https://user-images.githubusercontent.com/2870104/29094757-c7b66974-7c8e-11e7-875c-06f3a4e5d3f0.png)

The all containing headline is "Topology" and that makes total sense. In my eye statistics is part of device (topology'ly speaking) or it needs to move out of the Topology section, which would also be okay with me. We'd need to test that.